src/containers/GalacticPropertiesComboContainer.jsx

```diff
@@ -0,0 +1,203 @@
+      xValueAccessor: 'distance',
+      yValueAccessor: 'color',
+      tooltipAccessors: ['distance', 'color'],
+      tooltipUnits: ['Billion Ly', 'Flux ratio'],
+      tooltipLabels: ['Distance', 'Color'],
+      xDomain: [0, 16],
+      yDomain: [0, 2],
+    };
+  };
+
+  getDataObjects = (data, multiple) => {
+    if (multiple) {
+      return data.map(targetData => {
+        if (!targetData.objects) return null;
+        return targetData.objects;
+      });
+    }
+    return isArray(data.objects) ? data.objects : [];
+  };
+
+  userGalacticPropertiesCallback = data => {
+    console.log({ data }); // eslint-disable-line no-console
+  };
+
+  setActiveProperty = property => {
+    const { properties } = this.state;
+    const loadOptions =
+      property.name === 'Brightness vs Distance'
+        ? this.getBrightnessOptions()
+        : this.getColorOptions();
+    this.setState(prevState => ({
+      ...prevState,
+      navItems: this.generateNavItems(properties, property),
+      options: loadOptions,
+      updateDomain: true,
+    }));
+  };
+
+  generateNavItems(properties, activeScramble) {
+    return properties.map(property => {
+      const { name, color } = property;
+      return {
+        leftAvatar: (
+          <span>
+            <ScatterPlot style={{ fill: color }} />
+            <span className="screen-reader-only">{name}</span>
+          </span>
+        ),
+        primaryText: name,
+        className: classnames(propertyItem, {
+          [linkActive]: activeScramble.name === property.name,
+        }),
+        onClick: () => this.setActiveProperty(property),
+      };
+    });
+  }
+
+  handleDomainUpdatedResponse = () => {
+    this.setState(prevState => ({
+      ...prevState,
+      updateDomain: false,
+    }));
+  };
+
+  render() {
+    const { data, navItems, updateDomain, options } = this.state;
+    const {
+      title,
+      xAxisLabel,
+      yAxisLabel,
+      xValueAccessor,
+      yValueAccessor,
+      tooltipAccessors,
+      tooltipUnits,
+      tooltipLabels,
+      xDomain,
+      yDomain,
+    } = options || {};
+
+    return (
+      <>
+        <NavDrawer
+          interactableToolbar
+          cardClasses={drawerContainer}
+          contentClasses={galacticPropertiesDrawerContent}
+          navItems={navItems}
+          toolbarTitle={title}
+        >
+          <div className="container-flex spaced">
+            <div className="col padded">
+              {/* <h2 className="space-bottom">{title}</h2> */}
+              <GalacticProperties
+                className="color-brightness-vs-distance-combo"
+                {...{
+                  data,
+                  options,
+                  xAxisLabel,
+                  yAxisLabel,
+                  xValueAccessor,
+                  yValueAccessor,
+                  tooltipAccessors,
+                  tooltipUnits,
+                  tooltipLabels,
+                  updateDomain,
```

Comment:
And so if you made the changes described above in the GalacticProperties component you wouldn't pass this updateDomain bool through because GalacticProperties would just be responding to other props changing.